tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Process/ProcessStartCommon.cs

```diff
@@ -0,0 +1,83 @@
+// <copyright file="ProcessStartCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Process
+{
+    internal static class ProcessStartCommon
+    {
+        internal const IntegrationId IntegrationId = Configuration.IntegrationId.Process;
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(ProcessStartCommon));
+        internal const string OperationName = "command_execution";
+        internal const string ServiceName = "command";
+        internal const int MaxCommandLineLength = 4096;
+
+        internal static Scope CreateScope(ProcessStartInfo info)
+        {
+            if (info != null)
+            {
+                // In .NET framework, we need to have the UseShellExecute property set to false in order to use environment variables.
+#if !NETCOREAPP
```

Comment:
```suggestion
#if NETFRAMEWORK
```

I think you want `NETFRAMEWORK` here instead of `!NETCOREAPP`, otherwise the first `CreateScope()` will be compiled into the assembly targeting .NET Standard 2.0, which is used when the runtime is .NET Core < 3.1.

Reply:
Good point!